frontend/src/modules/colorLabel/views/colorLabelPopupView.js

```diff
@@ -0,0 +1,97 @@
+define([
+    'core/origin',
+    'core/views/originView'
+], function(Origin, OriginView) {
+
+    var ColorLabelPopUpView = OriginView.extend({
+
+        className: 'colorlabel',
+
+        events: {
+            'click .reset': 'onReset',
+            'click .apply': 'onApply',
+            'click .cancel': 'remove',
+            'click .color-item': 'onItemClick'
+        },
+
+        initialize: function(options) {
+            this.parentView = options.parentView;
+            OriginView.prototype.initialize.apply(this, arguments);
+        },
+
+        onItemClick: function(event) {
+            var $elm = $(event.currentTarget);
+            this.selected = $elm.data('colorlabel');
+            this.updateClasses();
+        },
+
+        updateClasses: function() {
+            this.$('.color-item').removeClass('selected');
+            this.$('[data-colorlabel="'+this.selected+'"]').addClass('selected');
+        },
+
+        onReset: function() {
+            var model = this.parentView.model;
+            this.selected = null;
+
+            model.save('_colorLabel', '', {
+                patch: true,
+                success: _.bind(this.onSuccess, this),
+                error: _.bind(this.onError, this)
+            });
+        },
+
+        onApply: function(event) {
+            if (!this.selected) return;
+            this.addItem();
+        },
+
+        preRender: function() {
+            this.selected = null;
+        },
+
+        postRender: function() {
+            var color = this.parentView.model.get('_colorLabel');
+            if (!color) return;
+            this.selected = color;
+            this.updateClasses();
+        },
+
+        applyOnParent: function() {
+            var color = this.selected;
+
+            if (this.selected) {
+                this.parentView.$el.attr('data-colorlabel', color);
```

Comment:
Now you chosen `null` as the falsey value, this whole function be condensed into one line:
```js
this.parentView.$el.attr('data-colorlabel', this.selected);
```
Reference: http://api.jquery.com/attr/#attr-attributeName-value.